LICENSE

```diff
@@ -1,5 +1,6 @@
                           DARMA Toolkit v. 1.0.0
```

Comment:
Shouldn't this line be also removed?

Reply:
This line needs to stay for copyright purposes.

Reply:
AFAIK this is required by copyright to be put at least in the license file (this is what Jonathan has used in Sandia copyright process).